src/components/app-field.js

```diff
@@ -110,21 +110,23 @@ class AppField extends React.PureComponent {
         ) : (
           <Table.Cell>
             {this.state.formOpen ? (
-              <div className='app-field-div'>
-                <Input
-                  styleName={
-                    isBrowser
-                      ? 'app-field-input-browser'
-                      : 'app-field-input-mobile'
-                  }
-                  type={'password'}
-                  value={this.state.password}
-                  onChange={this.handlePasswordChange}
-                  placeholder={this.state.passwordPlaceholder}
-                  error={this.state.passwordError}
-                  onKeyDown={this.handleKeyDown}
-                />
-                <Button.Group>
+              <div styleName={isBrowser ? 'app-field-div' : ''}>
+                <div>
+                  <Input
+                    styleName={
+                      isBrowser
+                        ? 'app-field-input-browser'
+                        : 'app-field-input-mobile'
+                    }
+                    type={'password'}
+                    value={this.state.password}
+                    onChange={this.handlePasswordChange}
+                    placeholder={this.state.passwordPlaceholder}
+                    error={this.state.passwordError}
+                    onKeyDown={this.handleKeyDown}
+                  />
+                </div>
+                <div styleName='app-field-div'>
```

Comment:
The buttons had no margin between them if `ButtonGroup` is used.
I have modified the code so that the buttons will have **default margin** between them and the design is device friendly.